zerver/migrations/0129_realm_emoji_drop_unique_constraint.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+import tempfile
+
+from django.conf import settings
+from django.db import migrations, models
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from zerver.lib.upload import get_bucket, upload_backend
```

Comment:
So, generally, importing things from `zerver` in migrations is dangerous, because if in a future version of Zulip, the code in `upload.py` changes, the migration will stop working for older users.  It would be better to copy the code into this file.